server/monitoring/metricsApp.ts

```diff
@@ -0,0 +1,28 @@
+import express from 'express'
+import promBundle from 'express-prom-bundle'
+
+const metricsMiddleware = promBundle({
+  autoregister: false,
+  buckets: [0.5, 0.75, 0.95, 0.99, 1],
+  httpDurationMetricName: 'http_server_requests_seconds',
```

Comment:
Didn't actually implement the superagent event handler in restClient because this project uses axios. Rather than rewrite the implementation might as well just switch to superagent in another PR and do it then.

Reply:
yes, thatd be great 👍